### ISSUE TITLE
Better support for named argument calling

### ISSUE THREAD

**Issue by domenic** [tabatkins/bikeshed#1758]
I'm starting to use the pattern in https://github.com/whatwg/infra/pull/325 and it's not working that great.

I have to use

```html
<a for=ReadableStream/create><var ignore>highWaterMark</var></a>
```

but I'd like to use something like

```
[=ReadableStream/create/|highWaterMark|=]
```

Any ideas?

**Comment by domenic** [comment]
It's also important that these `<var>`s not get highlighted by the var-highlighting JS when clicked.

**Comment by tabatkins** [comment]
Yeah that looks real bad.

How would you feel about `<arg>ReadableStream/create/highWaterMark</arg>`?

**Comment by domenic** [comment]
Works for me. I kinda like my markup shorthands instead of HTML tags, but I don't insist by any means, and I assume you're running out of character combinations :).

**Comment by tabatkins** [comment]
Yeah, I'm trying to limit grawlix in general; there's still plenty of space in the `[$..$]` style of tags, but short tagnames are nice too ^_^

**Comment by jyasskin** [comment]
Since [a call is likely to have several arguments scoped to the same algorithm](https://github.com/w3c/ServiceWorker/pull/1533/files#r486862412), I think we'll want to be able to default the `ReadableStream/create` part. As mentioned in https://github.com/whatwg/infra/issues/320#issuecomment-676736001,

```html
<call>[=ReadableStream/create=] with a <arg>highWaterMark</arg> of |value|</call>
```

might be a good syntax, where the algorithm name provides the default `for=` value for `<arg>`s. If `<call>` isn't in the cards for a while, we'll want to have something like

```html
<span arg-for="ReadableStream/create">[=ReadableStream/create=] with a <arg>highWaterMark</arg> of |value|</span>
```

**Comment by tabatkins** [comment]
Yup, definitely my thought too.
